content/docs/0.11.x/operate/user_management/openid_authentication/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Configuring Keptn OIDC
+description: Enable/Disable OpenID authentication and implementation details for the identity provider
+weight: 0
+url: /docs/0.11.x/reference/bridge/oauth/
```

Comment:
Was unsure about that, because sometimes it worked and sometimes not, depending on the compilation. Because the weight is below the weight of `Reference > Bridge > OpenID` I would expect that this is not preferred, but I guess this is not the case. @thisthat do you have a solution to add a redirect to another menu point, for the feature @johannes-b wants?
> Additionally, it would be great if we get a new menu entry in `Operate Keptn > User Management > Configuring Keptn OIDC` which then forwards to the page:   `Reference > Keptn Bridge > OpenID Authentication`

Reply:
Reading the [doc ](https://gohugo.io/content-management/urls/#how-hugo-aliases-work), I think this page should contain the `alias` annotation to be redirected to  `Reference > Keptn Bridge > OpenID Authentication`. Otherwise, you should move the content into this file.

Reply:
I tried it with the `alias` but it does not work.

I will now provide a cross-link that refers to the section of the Keptn Bridge.